source/features/table-input.css

```diff
@@ -0,0 +1,27 @@
+.rgh-add-table-popup {
+	width: 108px;
+	height: 108px;
+	padding: 4px;
+	z-index: 99;
+	display: grid;
+	grid-template: repeat(5, 1fr) / repeat(5, 1fr);
+	place-content: center;
```

Comment:
You're already setting width and height of both the container and the children, there's no need to specify another constraint
```suggestion
	display: flex;
	flex-wrap: wrap;
```

Reply:
Same for this. Flex isn't pita, it requires less code than before: c54d1e12

Also in the width wasn't applied in Chome:

<img width="374" alt="" src="https://user-images.githubusercontent.com/1402241/94379868-995f9680-00f8-11eb-8eb1-c9946694d234.png">

<img width="131" alt="" src="https://user-images.githubusercontent.com/1402241/94379866-9795d300-00f8-11eb-8dfc-078374d984b0.png">


That commit also fixed it